index.html

```diff
@@ -9,5 +9,16 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
+
+    <div class="title-container">
+        <img width="50px" height="50px" src="./assets/triangle.svg" alt="triangle">
+        <p>Front-End Tools</p>
```

Comment:
This is an important title, so remember to emphasize it by using tags like `h1`, `h2`.. etc